xrmocap/visualization/visualize_keypoints2d.py

```diff
@@ -0,0 +1,450 @@
+            Defaults to None.
+        background_img_list (Union[List[str], None], optional):
+            List of paths to images for background.
+            Defaults to None.
+        height (Union[int, None], optional):
+            Height of background. Defaults to None.
+        width (Union[int, None], optional):
+            Width of background. Defaults to None.
+        disable_tqdm (bool, optional):
+            Whether to disable tqdm progress bar.
+            Defaults to True.
+        logger (Union[None, str, logging.Logger], optional):
+            Logger for logging. If None, root logger will be selected.
+            Defaults to None.
+
+    Raises:
+        ValueError: Neither plot_points nor plot_lines is True.
+
+    Returns:
+        Union[np.ndarray, None]:
+            Plotted multi-frame image array or None.
+            If it's an array, its shape shall be
+            [n_frame, height, width, 3].
+    """
+    logger = get_logger(logger)
+    if not plot_points and not plot_lines:
+        logger.error('plot_points or plot_lines must be True.')
+        raise ValueError
+    n_frame = keypoints.get_frame_number()
+    n_person = keypoints.get_person_number()
+    n_kps = keypoints.get_keypoints_number()
+    mperson_colors = get_different_colors(
+        number_of_colors=n_person,
+        enable_random=False,
+        mode='rgb',
+        logger=logger)
+    if plot_points:
+        point_template = keypoints.get_keypoints()[0, 0, ..., :2]
+        point_palette_list = []
+        # construct palette for each person
+        for person_idx in range(n_person):
+            point_palette = PointPalette(
+                point_array=point_template,
+                name=f'point_palette_{person_idx}',
+                color_array=mperson_colors[person_idx],
+                logger=logger)
+            point_palette_list.append(point_palette)
+        # concat mperson's palette into one
+        if len(point_palette_list) > 1:
+            point_palette = PointPalette.concatenate(point_palette_list,
+                                                     logger)
+        else:
+            point_palette = point_palette_list[0]
+        mframe_point_data = keypoints.get_keypoints()[..., :2].reshape(
+            n_frame, n_person * n_kps, 2)
+        mframe_point_mask = keypoints.get_mask().reshape(
+            n_frame, n_person * n_kps)
+        # if only one person,
+        # use different colors for different points
+        if n_person == 1:
+            point_colors = get_different_colors(
+                number_of_colors=n_kps, mode='rgb', logger=logger)
+            point_palette.set_color_array(point_colors)
+    else:
+        point_palette = None
+        mframe_point_data = None
+        mframe_point_mask = None
+    if plot_lines:
+        limbs = get_limbs_from_keypoints(keypoints=keypoints, )
+        point_template = keypoints.get_keypoints()[0, 0, ..., :2]
+        conn = limbs.get_connections()
```

Comment:
Shall we add extra bone conn?

Reply:
To add new connections, it is recommended to edit `HUMAN_DATA_LIMBS` in human_data.py.